scripts/create_istio_system.sh

```diff
@@ -33,3 +33,14 @@ else
 fi
 istioctl manifest generate -i $ISTIO_NAMESPACE ${ISTIO_FILES[@]/#/-f } > /app/generated-manifest.yaml
 istioctl verify-install -i $ISTIO_NAMESPACE -f /app/generated-manifest.yaml
+
+if kubectl get namespace cattle-dashboards; then
+  kubectl delete configmap -n cattle-dashboards -l istio_dashboard=1
```

Comment:
Was thinking this could error out but should be fine

Reply:
This should not error. The namespace is present, that is checked in the line before. And if there are no configmaps that match this label, this does not do anything:

```
❯ kubectl get configmaps -n default -l notpresent=1
No resources found in default namespace.

❯ echo $?
0
```